.circleci/config.yml

```diff
@@ -50,9 +50,21 @@ jobs:
       - run: 
           command: |
             /bin/sh scripts/commitlint_range.sh
+  test_mongodb:
+    docker:
+      - image: circleci/mongo:latest
+    steps:
+      - checkout
+      - run:
+          command:
+            "mongo mongodb+srv://fhdatimedb-jjsjm.mongodb.net/test --eval 'db.serverStatus()'"
```

Comment:
I am thinking about we should put this link in the env to aovid security issue.

Reply:
should we use an environment variable?

Reply:
should be